specification/compute/resource-manager/Microsoft.Compute/stable/2022-07-02/DiskRP/diskRPCommon.json

```diff
@@ -449,6 +449,10 @@
     "SupportedCapabilities": {
       "type": "object",
       "properties": {
+        "diskControllerTypes": {
```

Comment:
please add an example for put with diskControllerTypes and patch with an update if its supported. Also check the characters.. looks like some bad characters are present in description